mix.exs

```diff
@@ -4,8 +4,8 @@ defmodule Placebo.MixProject do
   def project do
     [
       app: :placebo,
-      version: "2.0.0-rc.2",
-      elixir: "~> 1.8",
+      version: "2.1.0",
+      elixir: "~> 1.13",
```

Comment:
Hmm. I'm not sure if I agree with this change. If we require `1.13`, don't we break backwards compatibility with apps before `import Config` was available?

Reply:
Looks like the change I made to meck.exs works the same under both 1.13 and 1.8, so I'll revert that line.